docs/developer-docs/latest/setup-deployment-guides/file-structure.md

```diff
@@ -3,41 +3,100 @@ title: Project Structure - Strapi Developer Documentation
+│     │           ├──── <a href="/documentation/developer-docs/latest/development/backend-customization/controllers.html">controllers</a>
+│     │           ├──── <a href="/documentation/developer-docs/latest/development/backend-customization/policies.html#how-to-create-a-policy">policies</a>
+│     │           ├──── <a href="/documentation/developer-docs/latest/development/backend-customization/routing.html">routes</a>
+│     │           ├──── <a href="/documentation/developer-docs/latest/development/backend-customization/services.html">services</a>
+│     │           └ index.js
+│     ├──── <a href="/documentation/developer-docs/latest/development/backend-customization/models.html#components">components</a>
+│     │     └──── (category-name)
+│     │           ├ (componentA).json
+│     │           └ (componentB).json
+│     ├──── <a href="/documentation/developer-docs/latest/development/plugins-extension.html">extensions</a> <span class="token comment"># files to extend installed plugins</span>
+│     │     └──── (plugin-to-be-extended)
+│     │           └──── content-types
+│     │                 ├──── (content-type-name)
+│     │                 │     └ schema.json
+│     │                 └ <a href="/documentation/developer-docs/latest/developer-resources/plugin-api-reference/server.html">strapi-server.js</a>
+│     ├──── <a href="/documentation/developer-docs/latest/setup-deployment-guides/configurations/optional/middlewares.html">middlewares</a>
+│     │     └──── (middleware-name)
+│     │           ├ defaults.json
+│     │           └ index.js
+│     ├──── <a href="/documentation/developer-docs/latest/development/plugins-development.html">plugins</a> <span class="token comment"># local plugins files</span>
+│     │     └──── (plugin-name)
+│     │           ├──── admin
+│     │           │     └──── src
+│     │           │           └ <a href="/documentation/developer-docs/latest/developer-resources/plugin-api-reference/admin-panel.html">index.js</a>
+│     │           └──── <a href="/documentation/developer-docs/latest/developer-resources/plugin-api-reference/server.html">server</a>
+│     │           ├ package.json
+│     │           └ <a href="/documentation/developer-docs/latest/developer-resources/plugin-api-reference/server.html">strapi-server.js</a>
+│     └─── <a href="">policies</a>
+├ <a href="/documentation/developer-docs/latest/setup-deployment-guides/configurations/optional/environment.html#configuration-using-environment-variables">.env</a>
+└ .package.json
+  </code>
+</pre>
+  
+If the Strapi project was created with the [starter CLI](https://strapi.io/blog/announcing-the-strapi-starter-cli), the project structure looks like this:
 
 ```sh
 my-project
-  /frontend # starter folder
-  /backend  # template folder
-  /node_modules
+├─── frontend # starter folder
+├─── backend  # template folder
+└─── node_modules
 ```
 
 and the `backend` folder has the default structure described above.
```

Comment:
I'm not sure about this part 🤔 
- I wouldn't recommend starting a sentence, adding a code block in the middle, and then finish it. I think it would be better to write the whole sentence, and then add the code block.
- I'm afraid this part could easily be missed by most readers since the structure of the file doesn't indicate at all that both the regular project structure and the starter CLI project structure are shown/explained. Perhaps it would be better to move this piece of information at the beginning of the file, maybe in a note if possible. Or you could add 2 H2 titles but that may be a bit too much for that case. What do you think @pwizla ?

Reply:
I agree, I'd suggest just finishing the sentence and including the block below it.